sites/mosul/configs/openmrs/initializer_config/globalproperties/skip_location_datafilters.xml

```diff
@@ -0,0 +1,127 @@
+                Encounter-type based privilege filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_encTypePrivBasedConditionFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Encounter-type based condition filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_encTypePrivBasedEncounterFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Encounter-type based encounter filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_encTypePrivBasedEncounterTypeFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Encounter-type based encounter-type filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedVisitFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based visit filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedEncounterFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based encounter filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedObsFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based observation filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedPatientFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based patient filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedUserFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based user filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationBasedProviderFilter.disabled</property>
+            <value>true</value>
+            <description>
+                Location based provider filter. To enable, set value to false
+            </description>
+        </globalProperty>
+        <globalProperty>
+            <property>datafilter_locationFilter.disabled</property>
```

Comment:
Haven't gone in depth with the documentation of the data filter so am curious, is this a requirement for the appointments data filter to work? doesn't it affect the location filter that is responsible to restricting visits to specific locations?

Reply:
>is this a requirement for the appointments data filter to work? 

Yes, I think.  Default location filters if not disabled will prevent non-admin users from accessing locations.

The default filters that come with the module attend to only a specific O3 implementer's requirement. The rule of thumb is "exclude all default filters and add your own".

That said, when a requirement for restricting visits based on a location arises, we shall create a filter for that.